### PR TITLE
Replacement for #73

### PR DESCRIPTION
**Pull request by JuliaSprenger**

### CODE REVIEW

File: doc/tutorials/parallel.ipynb [hunk 1]
```diff
@@ -289,7 +289,7 @@
     "\n",
     "### Benchmark\n",
     "\n",
-    "Finally, let's see if `ProcessPoolExecutor` brings any speedup, compared to sequential processing, to Example 2."
+    "Finally, let's see if `ProcessPoolExecutor` brings any speedup, compared to sequential processing as demonstrated in Example 2."
```
Comment (dizcza) [inline doc/tutorials/parallel.ipynb]: well, sequential processing is not demonstrated in Example 2 (at the point the reader reads the sentence)
that's why I added benchmarks that demonstrate the speedup in Example 2

**Review by dizcza — COMMENTED** [review]
I agree with all the changes except the minor remark

File: doc/tutorials/parallel.ipynb (hunk 1 above)
Comment (JuliaSprenger) [reply doc/tutorials/parallel.ipynb]: Then I didn't get what you wanted to state previously. Let me make another suggestion.

Comment (JuliaSprenger) [reply doc/tutorials/parallel.ipynb]: I thought you wanted to state that in Example 2 you only used a single worker and want to compare to that... I now removed the reference to Example 2. Hopefully this removes also some potential for confusion.

**Pull request merged**